test/e2e/ctrcfg_test.go

```diff
@@ -162,16 +163,21 @@ func getMCFromCtrcfg(t *testing.T, cs *framework.ClientSet, ctrcfgName string) (
 // getValueFromCrioConfig jumps onto the node and gets the crio config. It then uses the regexKey to
 // find the value that is being searched for
 // regexKey is expected to be in the form `key = (\S+)` to search for the value of key
-func getValueFromCrioConfig(t *testing.T, cs *framework.ClientSet, node corev1.Node, regexKey string) string {
+func getValueFromCrioConfig(t *testing.T, cs *framework.ClientSet, node corev1.Node, regexKey string, getOverrideValue bool) string {
 	// get the contents of the crio.conf on nodeName
-	out := execCmdOnNode(t, cs, node, "cat", "/rootfs/etc/crio/crio.conf")
+	var out string
+	if getOverrideValue {
+		out = execCmdOnNode(t, cs, node, "cat", filepath.Join("/rootfs", ctrcfg.CRIODropInFilePathLogLevel))
```

Comment:
(Non-blocking, this works fine:  If “override” means “one particular override”, it would be a bit clearer to just have the caller provide a node-relative path argument (+ adding a constant for the `00-default` path) instead of a boolean.)

Reply:
Fixed